olm.sh

```diff
@@ -109,6 +109,17 @@ function main() {
     icon_mediatype=image/png
     "${YQ}" -i ".spec.icon[0].base64data |= (\"${icon_base64data}\")" "bundles/redhat-marketplace/${BUILD_VERSION}/manifests/${PACKAGE}.clusterserviceversion.yaml"
     "${YQ}" -i ".spec.icon[0].mediatype |= (\"${icon_mediatype}\")" "bundles/redhat-marketplace/${BUILD_VERSION}/manifests/${PACKAGE}.clusterserviceversion.yaml"
+
+    # To add related images after bundle creation
+    image=$("${YQ}" '.spec.install.spec.deployments[0].spec.template.spec.containers[0].image' "bundles/redhat-marketplace/${BUILD_VERSION}/manifests/${PACKAGE}.clusterserviceversion.yaml")
+    "${YQ}" -i ".spec.relatedImages[0].image |= (\"${image}\")" "bundles/redhat-marketplace/${BUILD_VERSION}/manifests/${PACKAGE}.clusterserviceversion.yaml"
+    "${YQ}" -i ".spec.relatedImages[0].name |= \"csi-provisioner\"" "bundles/redhat-marketplace/${BUILD_VERSION}/manifests/${PACKAGE}.clusterserviceversion.yaml"
```

Comment:
Can we also get image name and use it here instead of  hardcoded value?

Reply:
Yes, very good idea, let's do that! 👍